src/app/features/ledger/hooks/use-ledger-navigate.ts

```diff
@@ -42,7 +42,11 @@ export function useLedgerNavigate() {
         return navigate(RouteUrls.ConnectLedger, {
           replace: true,
           relative: 'route',
-          state: { tx: bytesToHex(psbt), inputsToSign },
+          state: {
+            tx: bytesToHex(psbt),
+            inputsToSign,
+            backgroundLocation: { pathname: RouteUrls.Home },
```

Comment:
I tested pre https://github.com/leather-wallet/extension/commit/6262267ff11fdeaa3f0d995fb5f286debcbbfd17 and we didn't need to do this before and I'm honestly not certain why we do now.

I have done local testing and confirmed I can send an ordinal from my ledger to my software wallet. Once it has arrived I will open this for review. 

Other flows like `send/btc/` don't need this `backgroundLocation` as they are page based rather than dialog based so I need to test them now. 

Likely I will need to add some other code to ensure they have the correct background but that is a lesser issue than not being able to send ordinals at all on Ledger

Reply:
> Other flows like send/btc/ don't need this backgroundLocation as they are page based rather than dialog based so I need to test them now.

MM, just read this. How can we set bg location dynamically?

Reply:
I've tested sending BTC and PSBT signing on Gamma from ledger and it's working. 

I have a hook from before that re-directs if there is no background location so we could use that or else try and have a set location it takes from global state.